content/booking/sbb_ticket_machine/index.en.md

```diff
@@ -0,0 +1,47 @@
+---
+draft: false
+title: "DB Website FIP (International)"
+description: "Booking information for FIP Tickets with a section in Germany."
+
+params:
+    classes:
+        first: "€6,90"
+        second: "€5,50"
+    fip_50: true
+    reservations: true
+    booking_link: "https://www.bahn.de/buchung/start?KL=2&ET=FIP_SONSTIGE"
+    additional_info_link: "https://int.bahn.de/en/offers/additional-services/seat-reservation"
+    type: "website"
+---
+
+{{% booking-section "fip_50" %}}
+**FIP 50 Tickets**
+
+Online FIP 50 Tickets for journeys within Germany and neighboring countries can be booked via Deutsche Bahn. The tickets must include at least one section within Germany. The FIP 50 ticket is issued for the entire route. The ticket is not valid in the country where the FIP ID was issued; an additional ticket is required there.
+
+Booking is only possible via the [link](https://www.bahn.de/buchung/start?KL=2&ET=FIP_SONSTIGE). Discounts do not need to be selected—they are applied automatically.
+
+FIP 50 Tickets can be purchased for the following countries, as long as a section in Germany is included:
+
+- Austria
+- Belgium
+- Czech Republic
+- Denmark
+- France
+- Germany
+- Italy (only Brenner route with Austria)
```

Comment:
Although this is not documented, tickets for the tri-national EC (also known as the ECE) from Frankfurt via Basel to Milan can also be purchased on this platform.

Reply:
A FIP50 ticket for the daily train from Frankfurt via Basel to Milan (and vice versa) is available online on the DB platform, but not the surcharge.

I bought the surcharge and the FIP50 ticket together for me and my wife at a DB Reisezentrum, which meant that we could sit together. I believe they were also able to sell the surcharge for the other Milan-Switzerland trains, but I would like to double-check.

Reply:
Ah now I understand. I was confused, because your comment had the sbb ticket machine file as reference :D

It still had the description from the db fip website because i didn't update the information in english.